regenerate.py

```diff
@@ -4,77 +4,64 @@
 
 import argparse
 import glob
-import itertools
 import logging
 import os
 import subprocess
 
 logger = logging.getLogger(__name__)
 LOGGING_LEVELS = {
-  'NOTSET': logging.NOTSET,
-  'DEBUG': logging.DEBUG,
-  'INFO': logging.INFO,
-  'WARNING': logging.WARNING,
-  'ERROR': logging.ERROR,
-  'CRITICAL': logging.CRITICAL,
+    'NOTSET': logging.NOTSET,
+    'DEBUG': logging.DEBUG,
+    'INFO': logging.INFO,
+    'WARNING': logging.WARNING,
+    'ERROR': logging.ERROR,
+    'CRITICAL': logging.CRITICAL,
 }
 DEFAULT_PANDOC_FLAGS = [
-  '-s',
-  '-H',
-  'header.html',
-  '-B',
-  'title.html',
+    '-s',
+    '--template=template.html',
+    '--base-header-level=2',
+    '--katex',
 ]
 
 
 def run(*args) -> str:
   logging.debug('Running command: %s', args)
-  result = subprocess.run(
-    args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf-8')
+  result = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf-8')
   logging.debug('\nStdout: %s\nStderr: %s', result.stdout, result.stderr)
   return result
 
 
-def GenerateHtml(pandoc_flags: List[str], doc_paths: List[str],
-                 post_paths: List[str]):
+def GenerateHtml(pandoc_flags: List[str], doc_paths: List[str], post_paths: List[str],
+                 build_dir: str):
```

Comment:
The `build_dir` name is a little confusing. Can we call it something else?

Reply:
I chose that because there's already an `output_dir`. What do you prefer? Maybe `site_root`?

Reply:
`site_root` SGTM.

Reply:
Changing throughout.